test/src/dgkernels/DGCoupledDiffusion.C

```diff
@@ -0,0 +1,133 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "DGCoupledDiffusion.h"
+
+// MOOSE includes
+#include "MooseVariable.h"
+#include "SystemBase.h"
+
+// libMesh includes
```

Comment:
This is IMO cpt. Obvious.

Reply:
Haha fine, although according to grep there are 195 captain obvious statements in the framework :-)

Reply:
Yes, those are leftovers from the time when libmesh was not using the `libmesh/` prefix. I guess it is time to remove them...